src/server.c

```diff
@@ -165,11 +170,16 @@ server_io_callback (GIOChannel *io, GIOCondition condition, gpointer user_data)
                output from restraintd is provided to the client through
                STDOUT using restraint_stdout_message in
                connections_write. */
-            if (!app_data->stdin)
+            if (app_data->stdin) {
```

Comment:
I'm not a fan of repeating the same piece of code in a number of place.  If you ahve to change something, the change
has to be made over and over again making it easy to miss something.